e2e/inference/README.md

```diff
@@ -31,28 +31,6 @@ To enable the interactive mode, the OpenVINO notebook CR needs to be created and
 1.	Click on the ```create Notebook``` option in this [link](https://github.com/red-hat-data-services/odh-deployer) from the web console and follow these [steps](https://github.com/openvinotoolkit/operator/blob/main/docs/notebook_in_rhods.md) to create the notebook CR.
```

Comment:
Please change the commit log as below:
e2e_inference: Remove RHODS GPU enabling workaround
The issue: https://github.com/opendatahub-io/odh-dashboard/issues/956 has been resolved by Accelerator Profile feature in RHODS 2.4.0 release.

Signed-off-by: vbedida79 [veenadhari.bedida@intel.com](mailto:veenadhari.bedida@intel.com)

Reply:
updated